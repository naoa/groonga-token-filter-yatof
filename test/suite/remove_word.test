register token_filters/yatof

table_create remove_words TABLE_HASH_KEY ShortText
load --table remove_words
[
{"_key": "and"},
{"_key": "or"}
]

tokenize TokenBigram "Hello and World" \
  --normalizer NormalizerAuto \
  --token_filters TokenFilterRemoveWord
