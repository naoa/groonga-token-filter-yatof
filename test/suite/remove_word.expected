register token_filters/yatof
[[0,0.0,0.0],true]
table_create remove_words TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
load --table remove_words
[
{"_key": "and"},
{"_key": "or"}
]
[[0,0.0,0.0],2]
tokenize TokenBigram "Hello and World"   --normalizer NormalizerAuto   --token_filters TokenFilterRemoveWord
[[0,0.0,0.0],[{"value":"hello","position":0},{"value":"world","position":2}]]
